2022/24xxx/CVE-2022-2422.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
@vcartman the file is in the wrong location. shouod be in 2022/2xxx iso 2022/24xxx